modules/core/shared/src/main/scala/lucuma/core/math/Parallax.scala

```diff
@@ -14,48 +19,48 @@ import spire.std.long._
 /**
   * Parallax stored as microarcseconds
   * Normally parallax is expressed in milliarcseconds but simbad reports them
-  * with a higher precision thus using microarcseconds gives us enough space
-  * Absolute allowed parallax values need to be less or equal thán 180 degrees
+  * with a higher precision thus using microarcseconds gives us enough space.
+  * Parallax values need to be in the interval [0°, 180°].
+  * 180° is a theoretical limit - actual astronomical values will be very small.
   */
-sealed abstract case class Parallax protected (μas: Quantity[Long, MicroArcSecond]) {
+sealed abstract case class Parallax protected (
+  μas: Quantity[Parallax.LongParallaxμas, MicroArcSecond]
+) {
   val mas: Quantity[Rational, MilliArcSecond] = μas.to[Rational, MilliArcSecond]
 
   override def toString: String =
     s"Parallax(${μas.show})"
 }
 
 object Parallax extends ParallaxOptics {
-  val MaxValue: Parallax = apply(648000000000L)
-  val MinValue: Parallax = apply(-648000000000L)
+  type Parallaxμas     = Interval.Closed[0, 648000000000L]
```

Comment:
At some point it might be good to define these singletons on `Angle` (i.e., `type Angle180µas = 648000000000L`) so we don't have so many magic numbers floating around. But this is fine for now.